mrjob/emr.py

```diff
@@ -2454,7 +2450,14 @@ def _pool_hash(self):
 
         if self._opts['bootstrap_mrjob']:
             things_to_hash.append(mrjob.__version__)
-        return hash_object(things_to_hash)
+
+        things_json = json.dumps(things_to_hash, sort_keys=True)
+        if not isinstance(things_json, bytes):
+            things_json = things_json.encode('utf_8')
```

Comment:
Uh, the canonical name of that encoding is "UTF-8".
The uppercase actually saves some nanoseconds..

Also, if you do dep yelp-bytes, this is the to-utf8 function.